build.gradle.kts

```diff
@@ -7,7 +7,7 @@ import java.time.format.DateTimeFormatter
 
 repositories {
     // provides dev versions!
-    maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/bootstrap")
+    //maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/bootstrap")
```

Comment:
If not needed then maybe let's remove them?

Reply:
This may still serves as reminder for the guys later. I did not modify this part.